Package.swift

```diff
@@ -29,10 +29,11 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.1.0")),
```

Comment:
For the Kitura repo we usually want to be more restrictive about the version since sometimes support for Swift versions is dropped in a minor version change which can cause compile-time breakage.